web/themes/new_weather_theme/templates/partials/daily-forecast-list-item.html.twig

```diff
@@ -20,24 +20,27 @@ only be a nighttime period.
         <span class="display-block">{{ periods[0].monthAndDay }}</span>
       </time>
 
-      {% for period in periods %}
-        {% include '@new_weather_theme/partials/daily-high-low.html.twig' with { 'period': period } %}
-      {% endfor %}
+      {% include '@new_weather_theme/partials/daily-high-low.html.twig' with { 'periods': periods } %}
     </div>
     <div class="grid-row grid-gap">
-      {% for period in periods %}
-        {% if period.isDaytime %}
-          {% set label = "Day" | t %}
-          {% set lineColor = "yellow" %}
-          {% set marginUnit = 3 %}
-        {% else %}
-          {% set label = "Night" | t %}
-          {% set lineColor = "indigo" %}
-          {% set marginUnit = 1 %}
-        {% endif %}
-        <div class="daily-forecast-summary-area grid-col-12 tablet:grid-col-6 margin-bottom-{{ marginUnit }} tablet:margin-bottom-1">
+        {% for period in periods %}
+            {% set marginUnit = 1 %}
+            {% set tabletCol = 6 %}
+            {%  if period.isOvernight %}
+                {% set label = "Overnight" | t %}
+                {% set lineColor = "indigo" %}
+                {% set tabletCol = 12 %}
+            {% elseif period.isDaytime %}
+                {% set label = "Day" | t %}
+                {% set lineColor = "yellow" %}
+                {% set marginUnit = 3 %}
+            {% else %}
+                {% set label = "Night" | t %}
+                {% set lineColor = "indigo" %}
+            {% endif %}
+        <div class="daily-forecast-summary-area grid-col-12 tablet:grid-col-{{ tabletCol }} margin-bottom-{{ marginUnit }} tablet:margin-bottom-1">
```

Comment:
@eric-gade I realized today my suggestion yesterday doesn't fully work for screens larger than mobile. We'll also need to account for `tablet:margin-bottom-1` on the overnight period, setting it to `tablet:margin-bottom-3` for that one.